client/client_test.go

```diff
@@ -221,11 +220,11 @@ func TestHttpTransport(t *testing.T) {
 		}),
 	}
 	client, err := New(cfg)
-	require.NoError(t, err)
+	assert.NoError(t, err)
```

Comment:
Cases like this will cause a panic because `client` will be `nil`.

The point of require is to stop executing because you know the rest of the test won't work anyway.

Reply:
I see, I didn't consider that we would branch on the `testify` assertions — that was naive. Thanks for being diligent, I'll close the PR.